std/manual.md

```diff
@@ -990,6 +990,132 @@ import { fib } from "./fib.wasm";
 console.log(fib(20));
 ```
 
+## Compiler API
+
+Deno supports runtime access to the built in TypeScript compiler. There are
+three methods in the `Deno` namespace that provide this access.
+
+### `Deno.compile()`
+
+This works similar to `deno run` on the command line, except that it doesn't
+execute any code. It takes up to three arguments, the `rootName`, optionally
```

Comment:
So it's more similar to `deno fetch` which downloads, caches and compiles without executing.